pkg/loadbalancer/experimental/lbmaps.go

```diff
@@ -90,6 +92,17 @@ type sourceRangeMaps interface {
 	DumpSourceRange(cb func(lbmap.SourceRangeKey, *lbmap.SourceRangeValue)) error
 }
 
+type maglevMaps interface {
+	/*
+		TODO: Perhaps it would be better to split UpdateMaglev into the following two methods:
```

Comment:
Why would this be better? It seems much worse to me.

Reply:
I would prefer keeping this logic behind `LBMaps` and it's fine if FakeLBMaps duplicates some of it.

Reply:
Sure, comment removed.